components/footer-link/footer-link.js

```diff
@@ -0,0 +1,29 @@
+const template = document.createElement("template");
+template.innerHTML = `
+  <link rel="stylesheet" href="./components/footer-link/footer-link.css">
+  <ul class="footer-link">
+    <li>
+      <a id="link" href="#">FAQ</a>
+    </li>
+  </ul>
+`;
+
+class FooterLinkComponent extends HTMLElement {
+  constructor() {
+    super();
+    const shadow = this.attachShadow({ mode: "open" });
+    shadow.appendChild(template.content.cloneNode(true));
+  }
+
+  static get observedAttributes() {
+    return ["href"];
+  }
+
+  attributeChangedCallback(name, oldValue, newValue) {
+    if (name === "href") {
+      this.shadowRoot.getElementById("link").setAttribute("href", newValue);
+    }
+  }
+}
+
+window.customElements.define("footer-link-component", FooterLinkComponent);
```

Comment:
#7 에 의해 한 단어가 아닌 컴포넌트는 뒤에 -component 를 붙이지 않도록 정했어요. 이 룰에 따라 custom element 이름과 class 이름을 변경해주실 수 있을까요?
해당 네이밍 룰이 이해가 안 된다면 또 말씀해주세요!

```suggestion
window.customElements.define("footer-link", FooterLink);
```

Reply:
네이밍 룰의 배경을 설명드리자면 component를 붙이는 것보다는 `-`를 붙여줌으로써 타 html 테그들과 분명히 구분하기 위해서 입니다. 그래서 이미 단어가 2개 이상이라 '-'이 사용되고 있다면 그대로 `meta-tag` 형식으로 쓰고 단어가 하나라면 `-`을 사용하기 위해서 뒤에 `-component`를 추가해주는겁니다!

Reply:
알려주신대로 수정하였습니다. 네이밍의 배경까지 친절하게 설명해주셔서 감사드립니다.